tools/run_tests/dockerize/docker_run_tests.sh

```diff
@@ -42,7 +42,11 @@ export PATH=$PATH:/usr/bin/llvm-symbolizer
 chown $(whoami) $XDG_CACHE_HOME
 
 mkdir -p /var/local/git
-git clone --recursive /var/local/jenkins/grpc /var/local/git/grpc
+git clone /var/local/jenkins/grpc /var/local/git/grpc
+
+# clone gRPC submodules
```

Comment:
I think adding an explanatory comment would be useful for future generations.
`# clone gRPC submodules, use data from locally cloned submodules where possible'